frontend/src/modules/scribble/components/LobbyID.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="lobby-id-section">
```

Comment:
Can you check the vue to be LobbyId.vue, and then this div should be the name of the component (lobby-id)